newIDE/app/src/AssetStore/ExtensionStore/index.js

```diff
@@ -122,6 +122,7 @@ export const ExtensionStore = ({
                     id="extension-search-bar"
                     value={searchText}
                     onChange={setSearchText}
+                    autoFocus
```

Comment:
Look at the typing of 
<img width="570" alt="image" src="https://user-images.githubusercontent.com/1280130/212469634-64dff47d-ac5a-4d96-97c9-950ac6d6c6a3.png">

You can enter:
```suggestion
                    autoFocus="desktop"
```

so that this will be autofocused, but just on desktop devices (on touchscreen like mobiles, we avoid to autofocus to avoid the virtual keyboard to always display when navigating :))

Reply:
Thanks. I realise I forgot to run the test/flow/format commands. Will add.